charts/plugin-br-pix-switch/templates/plugin-br-pix-switch/configmap.yaml

```
@@ -0,0 +1,64 @@
+  # Application
+  ENV_NAME: {{ .Values.pixSwitch.configmap.ENV_NAME | default "development" | quote }}
+  VERSION: {{ .Values.pixSwitch.configmap.VERSION | default "1.0.0-beta.1" | quote }}
+  SERVER_PORT: {{ .Values.pixSwitch.configmap.SERVER_PORT | default "4000" | quote }}
+  LOG_LEVEL: {{ .Values.pixSwitch.configmap.LOG_LEVEL | default "info" | quote }}
+
+  # gRPC / Protobuf
+  PROTO_PORT: {{ .Values.pixSwitch.configmap.PROTO_PORT | default "7001" | quote }}
+
+  # Auth
+  PLUGIN_AUTH_ENABLED: {{ .Values.pixSwitch.configmap.PLUGIN_AUTH_ENABLED | default "false" | quote }}
+  PLUGIN_AUTH_ADDRESS: {{ .Values.pixSwitch.configmap.PLUGIN_AUTH_ADDRESS | default "http://plugin-access-manager-auth.midaz-plugins.svc.cluster.local:4000" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)